mock-sdk/src/main/java/io/mosip/mock/sdk/service/SDKService.java

```diff
@@ -94,7 +94,7 @@ protected boolean isValidBIRParams(BIR segment, BiometricType bioType, String bi
 		case FACE:
 			break;
 		case FINGER:
-			if (!(bioSubType.equals("Unknown") || bioSubType.equals("Left IndexFinger")
+			if (!(bioSubType.equals("UNKNOWN") || bioSubType.equals("Left IndexFinger")
```

Comment:
should it be case sensitive? or can we make it case insensitive?

Reply:
In the spec we have

> For Finger: ["Left IndexFinger", "Left MiddleFinger", "Left RingFinger", "Left LittleFinger", "Left Thumb", "Right IndexFinger", "Right MiddleFinger", "Right RingFinger", "Right LittleFinger", "Right Thumb", "UNKNOWN"]
> For Iris: ["Left", "Right", "UNKNOWN"]
> For Face: No bioSubType